coderedcms/models/page_models.py

```diff
@@ -108,20 +108,64 @@ class CoderedPageMeta(PageBase):
     def __init__(cls, name, bases, dct):
         super().__init__(name, bases, dct)
         if 'search_db_include' not in dct:
-            cls.search_db_include = False
+            cls._search_db_include = False
```

Comment:
Let's completely remove the `search_db_*` fields. They were a poor man's scoring, but wagtail search actually sorts by relevancy now.